### DISCUSSION TITLE
how to configures

### DISCUSSION THREAD

**Discussion by BeneWilh** [LarsK1/hass_solvis_control#66] [General]
Hi @LarsK1,
thank you for this integration, but as a newbie I am not sseing any values in HA. Where and how to configure? Could you give some hints? Thx.

**Comment by LarsK1** [comment]
Hi @BeneWilh ,
did you install the plugin from HASS? Afterwards you need it to add as an Integration. Then type in the IP-Adress and the port and you're good to go...

**Reply by BeneWilh** [reply]
First of all, thank you for the great Integration, with your hint I got it running... maybe 3 lines of how-to in the readme would help also other newbies ;-)
Are you planing to integrate also more options? wooden burning stove? Heatpump?

**Reply by LarsK1** [reply]
Can you maybe explain, what you're excatly missing in the Readme?
